content/pt/case-studies/comcast/index.html

```diff
@@ -0,0 +1,4 @@
+---
+title: Comcast
+content_url: https://youtu.be/lmeFkH-rHII
```

Comment:
Why adding this page and the `comcast_logo` if is the same as the en content version?

Reply:
That's true. Deleted.